Cargo.toml

```diff
@@ -39,15 +39,15 @@ gluesql-core = { path = "./core", version = "0.15.0" }
 
 cli = { package = "gluesql-cli", path = "./cli", version = "0.15.0" }
 test-suite = { package = "gluesql-test-suite", path = "./test-suite", version = "0.15.0" }
-memory-storage = { package = "gluesql_memory_storage", path = "./storages/memory-storage", version = "0.15.0" }
-shared-memory-storage = { package = "gluesql-shared-memory-storage", path = "./storages/shared-memory-storage", version = "0.15.0" }
-sled-storage = { package = "gluesql_sled_storage", path = "./storages/sled-storage", version = "0.15.0" }
-json-storage = { package = "gluesql-json-storage", path = "./storages/json-storage", version = "0.15.0" }
-csv-storage = { package = "gluesql-csv-storage", path = "./storages/csv-storage", version = "0.15.0" }
-composite-storage = { package = "gluesql-composite-storage", path = "./storages/composite-storage", version = "0.15.0" }
-web-storage = { package = "gluesql-web-storage", path = "./storages/web-storage", version = "0.15.0" }
-idb-storage = { package = "gluesql-idb-storage", path = "./storages/idb-storage", version = "0.15.0" }
-redis-storage = { package = "gluesql-redis-storage", path = "./storages/redis-storage", version = "0.15.0" }
-mongo-storage = { package = "gluesql-mongo-storage", path = "./storages/mongo-storage", version = "0.15.0" }
-parquet-storage = { package = "gluesql-parquet-storage", path = "./storages/parquet-storage", version = "0.15.0" }
+gluesql_memory_storage = { package = "gluesql_memory_storage", path = "./storages/memory-storage", version = "0.15.0" }
+gluesql-shared-memory-storage = { package = "gluesql-shared-memory-storage", path = "./storages/shared-memory-storage", version = "0.15.0" }
```

Comment:
now workspace names are identical to package names so we can remove `package = "gluesql_memory_storage"` like additional package fields